walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,113 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
+  /** 
+   * Requires that collection of events are sorted in order of meeting start time. 
+   */
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Looks like most of your code is 2-space tabbed, but this is 4-space tabbed, accidentally aligning it with its implementation.